include/capstone/riscv.h

```diff
@@ -54,7 +54,7 @@ typedef struct cs_riscv {
 	// or 0 when instruction has no operand.
 	uint8_t op_count;
 	// Does this instruction need effective address or not.
-	uint8_t need_effective_addr;
+	bool need_effective_addr;
```

Comment:
cool, now just move `need_effective_addr` after operands[], or before `op_count`, then update Python binding for this new field.

Reply:
done

Reply:
and riscv.py update?

Reply:
updated, Please check, I am not sure the Python code is correct.

Reply:
well done, that is correct.

the final request is to add `test_riscv.py` to https://github.com/aquynh/capstone/blob/next/bindings/python/Makefile#L74, so `make check` execute this.